client/packages/common/src/ui/icons/Alert.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
 
-export const AlertIcon = (props: SvgIconProps): JSX.Element => {
+export const AlertIcon = (
+  props: SvgIconProps & { fill?: string }
+): JSX.Element => {
+  const { fill = 'none', ...rest } = props;
   return (
     <SvgIcon
-      {...props}
+      {...rest}
       viewBox="0 0 24 24"
       stroke="currentColor"
       strokeWidth="2"
       strokeLinecap="round"
       strokeLinejoin="round"
-      style={{ fill: 'none' }}
+      style={{ fill }}
```

Comment:
had to set the fill to a solid colour so that the alert icon shows over the top of the sync icon when the drawer is collapsed